src/structures/Role.js

```diff
@@ -233,6 +233,10 @@ class Role extends Base {
    * @param {RoleResolvable} role Role to compare to this one
    * @returns {number} Negative number if this role's position is lower (other role's is higher),
    * positive number if this one is higher (other's is lower), 0 if equal
+   * @example
+   * // Compare the position of a role to another
+   * const roleCompare = role.comparePositionTo(otherRole);
+   * if (roleCompare === 1) console.log(`${role.name} is higher than ${otherRole.name}`);
```

Comment:
```suggestion
   * if (roleCompare >= 1) console.log(`${role.name} is higher than ${otherRole.name}`);
```

Reply:
Wow, that's odd. It was reverted in https://github.com/discordjs/discord.js/pull/9252/commits/6cdf80ad59d0ebcb4dc06d0bb9dbc2077e04e5dd after my suggestion which did not modify it (https://github.com/discordjs/discord.js/pull/9252#discussion_r1141573994). It was manually reverted back.